### DISCUSSION TITLE
Link + Icon

### DISCUSSION THREAD

**Discussion by DiegodeMiguel** [Telefonica/mistica-design#527] [Proposals]
### Context
From TCX design team we think that we could evolution the link option into link + icon to have more relevance in some cases.

- [x] Check to confirm

When will I receive a reply?
We usually review component request Tuesdays at 16:00 (CEST), after the meeting, we will reply in this component request.

---
Here an example of the possible evolution component:

<img width="742" alt="image" src="https://user-images.githubusercontent.com/106736588/171597065-2d47d0e7-26a9-44eb-876f-74e9636acca8.png">

**Comment by yceballost** [comment]
We are going to proceed to include this petition to Mística, we need to evolve specs and then create tickets for development teams. 

We can't confirm a date for the release yet
